docs/cli/ory-tunnel.md

```diff
@@ -12,7 +12,7 @@ To improve this file please make your change against the appropriate "./cmd/*.go
 
 ## ory tunnel
 
-Tunnel Ory on a subdomain of your app or a seperate port your app's domain
+Tunnel Ory on a subdomain of your app or a separate port your app's domain
```

Comment:
Since the CLI docs are autogenerated, this will be changed to "separate" the next time CLI docs are generated until it's fixed in the code.

Reply:
Ah I just used search&replace.